stdlib/2/__builtin__.pyi

```diff
@@ -943,7 +947,10 @@ class list(MutableSequence[_T], Generic[_T]):
     def remove(self, __value: _T) -> None: ...
     def reverse(self) -> None: ...
     if sys.version_info >= (3,):
-        def sort(self, *, key: Optional[Callable[[_T], Any]] = ..., reverse: bool = ...) -> None: ...
+        @overload
+        def sort(self: List[_LT], *, key: None = ..., reverse: bool = ...) -> None: ...
+        @overload
+        def sort(self, *, key: Callable[[_T], _LT], reverse: bool = ...) -> None: ...
```

Comment:
Since you are only using one instance of `_LT` in each overload, the type var is redundant. Just using `_SupportsLessThan` should work.

Reply:
Thanks for pointing this out @srittau , I will change that in my next commit.